public/stylesheets/rtl/main.css

```diff
@@ -1,140 +1,100 @@
-/* /////////////////////////////////////////////////////
```

Comment:
I wouldn't worry about the /rtl/main.css at this point.

This gets built automatically via some of the i18n framework, but we don't actually have users in RTL languages at this point. When we do, I'm sure we'll find plenty of RTL issues, and we can start figuring out the build quirks then.